hack/build-index-image.sh

```diff
@@ -35,22 +35,60 @@ function create_index_image() {
   BUNDLE_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${BUNDLE_REGISTRY_IMAGE_NAME}:${CURRENT_VERSION}"
   INDEX_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${INDEX_REGISTRY_IMAGE_NAME}:${CURRENT_VERSION}"
 
-  INDEX_IMAGE_PARAM=
-  if [[ -n ${2} ]]; then
-    PREV_INDEX_IMAGE="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${INDEX_REGISTRY_IMAGE_NAME}:${PREV_VERSION}"
-    INDEX_IMAGE_PARAM=--from-index="${PREV_INDEX_IMAGE}"
-  fi
-
   podman build -t "${BUNDLE_IMAGE_NAME}" -f bundle.Dockerfile --build-arg "VERSION=${CURRENT_VERSION}" .
   podman push "${BUNDLE_IMAGE_NAME}"
 
   # Referencing the unstable bundle digest in the index image, rather than the floating tag, to avoid
   # unalignment between cached index image and fetched bundle image.
   BUNDLE_IMAGE_NAME=$("${PROJECT_ROOT}/tools/digester/digester" --image "${BUNDLE_IMAGE_NAME}")
 
-  # shellcheck disable=SC2086
-  ${OPM} index add --bundles "${BUNDLE_IMAGE_NAME}" ${INDEX_IMAGE_PARAM} --tag "${INDEX_IMAGE_NAME}" -u podman --mode semver
+
+  # File-Based Catalog handling
+  cd "${OUT_DIR}"
```

Comment:
This will work. But I'm not sure I'm happy with `cd` and `cd` back. What do you think about placing all of this in a new function, then call it like this?

```shell
(cd "${OUT_DIR}" && my_new_function)
```

Reply:
valid point.
updated, thanks.